packages/website/src/examples/layout/Table/TablePredicate.demo.tsx

```diff
@@ -1,6 +1,7 @@
-import {ColumnDef, createColumnHelper, Table, useTable} from '@coveord/plasma-mantine';
+import {Badge, ColumnDef, createColumnHelper, Group, Table, useTable} from '@coveord/plasma-mantine';
 import {faker} from '@faker-js/faker';
 import {useMemo} from 'react';
+import {TablePredicateProps} from '../../../../../mantine/dist/esm/components/table/table-predicate/TablePredicate';
```

Comment:
This import looks weird to me, it should be part of the `@coveord/plasma-mantine` import

Reply:
fixed